### ISSUE TITLE
Kubernetes Follow Logs command should invoke in new terminal

### ISSUE THREAD

**Issue by bnookala** [Azure/vscode-kubernetes-tools#273]
The shared terminal might be running another task. Additionally, since it's possible for the command to be run a long running task in the shared terminal, which could block future tasks. We should change the follow logs command to use a new terminal.

**Comment by kameshsampath** [comment]
since I did the same for the Kube events, I can take this up once the other PR is merged.

**Comment by bnookala** [comment]
@kameshsampath thanks for the offer to help - I'm almost finished with a solution to this :)

**Comment by kameshsampath** [comment]
no worries.
